apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -576,6 +576,18 @@ API.v1.addRoute(
 				return API.v1.failure('Team not found');
 			}
 
+			const room = Rooms.findOneById(teamInfo.roomId);
+
+			if (!teamInfo) {
+				return API.v1.failure('Room not found');
+			}
```

Comment:
```suggestion
			if (!teamInfo) {
				return API.v1.failure('Room not found');
			}

			const room = Rooms.findOneById(teamInfo.roomId);
```
Since we're using using information from `teamInfo` to search for the room, we should check if `teamInfo` is not undefined _before_ searching for the room.

Reply:
updated!